src/together/types/finetune.py

```diff
@@ -150,8 +150,14 @@ class FinetuneRequest(BaseModel):
     n_epochs: int
     # training learning rate
     learning_rate: float
+    # learning rate scheduler type and args
+    lr_scheduler: FinetuneLRScheduler | None = None
     # learning rate warmup ratio
     warmup_ratio: float
+    # max gradient norm
+    max_grad_norm: float | None = None
+    # weight decay
+    weight_decay: float | None = None
```

Comment:
I understand why it can be None for responses (for example, older jobs without the argument), but can it be just `float` in case of requests?

Reply:
Good point, changed!